MembershipApp/templates/inventory/DisplayCategory.html

```diff
@@ -0,0 +1,119 @@
+    <title>{{name}}</title>
+{% endblock %}
+
+{% block body %}
+
+    <h3 style="margin-left:1%">{{ header }}</h3>
+
+{% if category.items_type == "non_electrical_items" %}
+    <a href="{% url 'inventory:add_item' category.id %}" class="btn btn-warning btn-sm btn-pad" role="button" style="margin-left: 1%" >Add</a><br><br>
+{% else %}
+    <a href="{% url 'inventory:add_electrical_item' category.id %}" class="btn btn-warning btn-sm btn-pad" role="button"  style="margin-left: 1%" >Add</a><br><br>
+{% endif %}
+
+<form class="form-inline md-form mr-auto mb-4" style="margin-left: 1%">
+  <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search" >
+  <button class="btn btn-outline-warning btn-rounded btn-sm my-0" type="submit" style="width: 100px">Search</button>
+</form>
+
+    {% if category.items_type == "non_electrical_items" %}
+
+          <table class="table table-hover">
+            <thead>
+              <tr>
+                  <th>Member</th>
+                  <th>Title</th>
+                  <th>Description</th>
+                  <th>Width (m)</th>
+                  <th>Height (m)</th>
+                  <th>Length (m)</th>
+                  <th>Quantity</th>
+                  <th>Unit Price ($)</th>
+                  <th>Picture</th>
+                <th></th>
+              </tr>
+            </thead>
+
+            <tbody>
+
+              {% for item in items %}
+
+              <tr>
+                  <td>{{ item.member }}</td>
+                  <td>{{ item.title }}</td>
+                  <td>{{ item.description }}</td>
+                  <td>{{ item.width }}</td>
+                  <td>{{ item.height }}</td>
+                  <td>{{ item.length }}</td>
+                  <td>{{ item.quantity }}</td>
+                  <td>{{ item.unit_price }}</td>
+                    {% if item.picture %}
+                  <td><a href="{{ item.picture.url }}" title="click here to see the full sized image"><img src="{{ item.picture.url }}" style="width:50px;height:50px;" ></a></td>
+                    {% else %}
+                        <td><i> no pic </i></td>
+                    {% endif %}
+                  <td>
+                      <a href="{% url 'inventory:edit_item' item.pk %}" class="btn btn-warning btn-sm" role="button" aria-pressed="true" >Edit</a>
+                      <a href="{% url 'inventory:delete_item' item.pk%}" class="btn btn-danger btn-sm" role="button" aria-pressed="true" >Delete</a>
+                  </td>
+              </tr>
+
+              {% endfor %}
+            </tbody>
+          </table>
+    {% elif category.items_type == "electrical_items"  %}
+              <table class="table table-hover">
+            <thead>
+              <tr>
+                  <th>Member</th>
+                  <th>Title</th>
+                  <th>Description</th>
+                  <th>Width (m)</th>
+                  <th>Height (m)</th>
+                  <th>Length (m)</th>
```

Comment:
i see some code repetition here. Since the electrical item and the normal item share lots of these fields, i think you can share the code and only have an extra if statement for when the item is electrical.